nexus/reconfigurator/planning/src/blueprint_editor/sled_editor.rs

```diff
@@ -587,6 +588,9 @@ impl ActiveSledEditor {
                     .remove_mupdate_override
                     .finalize(),
                 host_phase_2: self.host_phase_2.finalize(),
+                // This will come later
```

Comment:
Maybe mark this with a `TODO:` and a slightly longer description? I'd also probably open an issue if it's going to take a bit to implement.